packages/terra-card/src/Card.jsx

```diff
@@ -12,20 +12,31 @@ const propTypes = {
    * Child Nodes
    */
   children: PropTypes.node,
+  /**
+   * An option used in order to change to a different style of card for different use cases
```

Comment:
Let list the options in the description so consumers know what variants there are.

Reply:
Good call, these are what generate the doc site props table, aren't they?